Test/tests.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kan-Elbash WEB</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <input type="text" name="searchMovie">
+      <button type="submit" name="submit">serach</button>
+    </form>
+
+    <div class="infoMovie"></div>
+    <div class="giphyMovie"></div>
```

Comment:
you need to import the QUnit scripts. See http://qunitjs.com/ 'Getting Started' section.